pkgs/applications/misc/cotp/default.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  AppKit,
+  libxcb,
+}:
+rustPlatform.buildRustPackage rec {
+  name = "cotp";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "replydev";
+    repo = "cotp";
+    rev = "v${version}";
+    sha256 = "sha256-SbS+ch7/45kZb49jW2mnRWQruLrfrNfeZFqEPHQKGUg=";
+  };
+
+  cargoSha256 = "sha256-2lIR3K4/hr4XSmNGFd/dhwoFOtEB9KSnUrZkcaCyc9k=";
```

Comment:
```suggestion
    shaHash = "sha256-SbS+ch7/45kZb49jW2mnRWQruLrfrNfeZFqEPHQKGUg=";
  };

  cargoHash = "sha256-2lIR3K4/hr4XSmNGFd/dhwoFOtEB9KSnUrZkcaCyc9k=";
```

Reply:
I think this changes was what made the evaluation fail (complained about `shaHash`). Reverting to previous state to confirm.

Reply:
Looks like everything is passing now!

Reply:
Apologies, I meant just `hash` not `shaHash`. That's just a nitpick and is not really required to change these, though.